### ISSUE TITLE
Is the site done?

### ISSUE THREAD

**Issue by Brandons404** [routman/publicnote.com#10]
I use this site on a daily basis as an easy way to transfer links/text and other things between mobile and desktop.

When trying to access the site this morning I noticed a pig emoji. Nothing else. Whats the deal? Is the site going back up soon? I would hate to see it go.

**Comment by routman** [comment]
Yes it's done.  It occurred to me that there was not a single good reason for me to continue paying hundreds of dollars each year to host it when there are far more important things to spend that money on, like college savings for my kids.  Especialy when you consider the type of crowd the site attracts.

**Comment by mgeirin** [comment]
Hi, would you consider creating a way for people to extract the content of a key? Thanks.

**Comment by Brandons404** [comment]
Makes sense. Well thanks for hosting as long as you did.. I enjoyed it while it was up.

**Comment by teaonvenus** [comment]
If there's any way to access old data would love to know about it, my community had a mega-list of funny band names going for a while. If there's a way that wasn't purged you would be our hero haha.

Regardless, thanks for maintaining it as long as you did. It was a great little tool, super helpful for quickly sharing and collaborating lists and links. Shame to see some of the attitudes that turned up. Best of luck with all.
